content/en/tracing/profiler/getting_started.md

```diff
@@ -213,33 +222,29 @@ The Datadog Profiler requires Go 1.12+. To begin profiling applications:
 
 **Note**:
 
-- By default only the CPU and Heap profile are enabled. Use [profiler.WithProfileTypes][4] to enable additional [profile types][5].
-
```

Comment:
I agree with adding the sample code comments, but I think this note, particularly the links in it, is useful and shouldn't be removed.

Reply:
@kayayarai ok, I reverted it again for now: 38d765289ffebe88d4f32708baa8fc7802448feb

That being said, I'd love to hear your thoughts on documenting each individual configuration option in this repository vs the [documentation of the package itself](https://pkg.go.dev/gopkg.in/DataDog/dd-trace-go.v1/profiler)?

It's also a little odd that the `WithProfileTypes` option is not listed in the table like the other options we're describing (my fault, not sure why I did this).

I don't think this discussion should hold up this PR, but I'd love to make the Go profiler docs better in general so if you have any thoughts please let me know. I'm happy to do some follow-up PRs.

Reply:
I'm about to go through the Profiling section of the docs and identify gaps and inconsistencies and places we can improve. I'll add this matter to that list. Thank you!

Reply:
@kayayarai awesome, thank you so much!